river/XwaylandView.zig

```diff
@@ -269,3 +276,14 @@ fn handleRequestFullscreen(listener: *wl.Listener(*wlr.XwaylandSurface), xwaylan
     self.view.pending.fullscreen = xwayland_surface.fullscreen;
     self.view.applyPending();
 }
+
+/// Some X11 clients get stuck in minimized state when they lose focus, which
+/// cause black screen, therefore we just disable it when a request is received
+/// as river doesn't support it.
+fn handleRequestMinimize(
```

Comment:
This comment and the code below do not match the findings and fix in the linked sway issue/PR...